### PR TITLE
azure rbac provider docs

### PR DESCRIPTION
**Pull request by vutran01**
## PR Checklist

* [ ] Explain your changes, so people looking at the PR know *what* and *why*, the code changes are the *how*.
Azure RBAC provider enables user to check permission on each resource. this is the documentation associated with this provider.

image

example workbooks: https://ms.portal.azure.com/#blade/AppInsightsExtension/UsageNotebookBlade/ComponentId/Azure%20Monitor/ConfigurationId/%2Fsubscriptions%2F8980832b-9589-4ac2-b322-a6ae6a97f02b%2Fresourcegroups%2Fvutest1%2Fproviders%2Fmicrosoft.insights%2Fworkbooks%2Fbabb6a1e-84ff-4477-a9cd-643b7c0d702e/Type/workbook/WorkbookTemplateName/rbac-permission

### If adding or updating templates:
* [ ] post a screenshot of templates and/or gallery changes
* [ ] ensure your template has a corresponding gallery entry in the gallery folder
* [ ] If you are adding a new template, add your team and template/gallery file(s) to the CODEOWNERS file. CODEOWNERS entries should be teams, not individuals
* [ ] ensure all steps have meaningful names
* [ ] ensure all parameters and grid columns have display names set so they can be localized
* [ ] ensure that parameters id values are unique __or they will fail PR validation__ (parameter ids are used for localization)
* [ ] ensure that steps names are unique __or they will fail PR validation__ (step names are used for localization)
* [ ] grep `/subscription/` and ensure that your parameters don't have any hardcoded resourceIds __or they will fail PR validation__
* [ ] remove `fallbackResourceIds` and `fromTemplateId` fields from your template workbook __or they will fail PR validation__

### CODE REVIEW

File: Documentation/DataSources/DataSources.md [hunk 1]
```diff
@@ -149,6 +150,29 @@ To avoid automatically making calls to un-trusted hosts when using templates, th
...
+
+**Array:**
+```
+["microsoft.insights/workbooks/read", "microsoft.insights/workbooks/write"]
+```
+
+**Action list:**
+```
+{
+  "actions": [
```
Comment (yves-chan) [inline Documentation/DataSources/DataSources.md:169]: does this json object usually need to have anything else other than actions array? I wonder if we can just omit this part for simplicity and have users only use the string or Array as the input?

Comment (vutran01) [reply Documentation/DataSources/DataSources.md:169]: that's a good point and until i noticed that https://docs.microsoft.com/en-us/rest/api/authorization/permissions/list-for-resource supported this and i added it. we could just remove it from the docs for now

**Review by yves-chan — APPROVED** [review]
LGTM

File: Documentation/DataSources/DataSources.md (hunk 1 above)
Comment (yves-chan) [reply Documentation/DataSources/DataSources.md]: Do we support notActions for the RBAC provider in workbooks?

**Pull request merged**